### DISCUSSION TITLE
SAE update auch online möglich ?

### DISCUSSION THREAD

**Discussion by DrMabuse100** [camueller/SmartApplianceEnabler#321] [Installation und Betrieb]
Hallo ist es möglich den SAE auch online abzudaten ?
Also so das ich in der console nur den update befehl eingebe ?

Danke freundlicher Gruß

**Comment by camueller** [comment]
Wenn ich mir Deine bisherigen Postings so ansehe, habe ich den Eindruck, dass Du nicht in der Lage bist, die Dokumentation zu lesen/zu verstehen.
Da kann ich leider nicht helfen.

**Comment by DrMabuse100** [comment]
Hallo, doch habe ich. Aber wie ich das sehe ist kein 100%iges online update möglich sondern es muss eine datei auf den rasppery.

**Comment by DrMabuse100** [comment]
Danke für die Info
